fiftyone/operators/executor.py

```diff
@@ -146,12 +147,18 @@ def execute_operator(operator_uri, ctx=None, **kwargs):
     """
     request_params = _parse_ctx(ctx=ctx, **kwargs)
 
-    result = asyncio.run(
-        execute_or_delegate_operator(
-            operator_uri, request_params, exhaust=True
-        )
+    coroutine = execute_or_delegate_operator(
+        operator_uri, request_params, exhaust=True
     )
-    result.raise_exceptions()
+
+    if foc.is_notebook_context():
+        # Notebooks already have event loops running, so we must execute there
+        loop = asyncio.get_event_loop()
+        loop.create_task(coroutine)
+        result = None  # @todo can we await result here?
```

Comment:
@benjaminpkane any asyncio wizardry here? Ideally we want to be able to await the tasks completion so we can return its `result()` to the user.

Some operators may perform computation and the user may want to access the data from the returned result object.

Reply:
Will take a look today 👍

Reply:
I think you could do
```python
    loop = asyncio.get_event_loop()
    result = loop.run_until_complete(coroutine)
# later ...
result.raise_exceptions()
return result
```

Reply:
awaiting ben's comments as a more async savvy person though 👍🏼